src/worker/triggers/keyword.ts

```diff
@@ -1,17 +1,21 @@
-import * as tf from '@tensorflow/tfjs'
-import { Tensor } from '@tensorflow/tfjs'
-import { RingBuffer } from './ringBuffer'
-import { CommandModels, EventType, SpeechConfig, SpeechContext, SpeechEvent } from './types'
-import { SpeechProcessor } from './speechProcessor'
+import RingBuffer from '../RingBuffer'
```

Comment:
_pushes glasses up_ Teeeeechnically, this isn't a "trigger"—triggers send the `Activate` event, which starts active listening in the pipeline and sends future audio to recognizer components. This is a recognizer (it produces speech transcripts, even though they're just from a limited vocabulary)—as we fill out the pipeline, I think we'll probably want separate folders for triggers, recognizers, and nlu components.

Reply:
I'm fine with that.

Reply:
What if I just rename the folder "processors"?

Reply:
I went ahead and renamed it. We can always separate triggers from recognizers later, but there's a nice symmetry between this and the variable in the worker.